index.html

```diff
@@ -1,48 +1,122 @@
+                </div>
+            </dl> 
+                <br>
+                <br>
+            <dl>   
+                <h2 class="nombres"><dt>Guardia Ortodoxa</dt></h2>
+                <div class="div2">
+                    <dd class="descrip">Una postura tradicional ortodoxa es aquella en la que el boxeador coloca su pie izquierdo más adelantado, teniendo su lado más débil cerca del oponente. Esta postura, ya que favorece el lado más fuerte, el derecho, es una de las más utilizadas en el boxeo, utilizada mayoritariamente por los diestros. La mano dominante, en este caso la derecha, se ubica protegiendo la sien. Mientras que la mano izquierda se ubica levemente adelantada.</dd>
+                    <img src="assets/Ortodoxa.jpg"; class="imagenes">
+                </div>
+                <div class="centerVideo">
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/hBxBHUUlxpw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen; class="videos"></iframe>
+                </div>
+            </dl>
+                <br>
+                <br>
+            <dl>  
+                <h2 class="nombres"><dt>Philly Shell</dt></h2>
+                <div class="div2">
+                    <dd class="descrip">Aquí el brazo más adelantado (izquierdo para un combatiente ortodoxo y derecha para un zurdo) se coloca a través del torso generalmente en alguna parte entre el ombligo y el pecho mientras que la mano del mismo brazo descansa en el lado opuesto del torso del combatiente. La mano trasera se pone en el lado de la cara (derecho para los combatientes ortodoxos y lado izquierdo para los zurdos). El hombro más adelantado se retrae adentro firmemente contra el lado de la cara.  Ejecutar a esta defensa un combatiente debe ser muy atlético y experimentado. Este estilo es tan eficaz para “contragolpear” porque permite que los combatientes evadan golpes mediante la rotación y descenso de su parte superior y generar ataques que desorientan del combatiente. Después de que el primer golpe confunde, la mano detrás está en posición perfecta para golpear a su oponente que está mal colocado.</dd>
+                    <img src="assets/Philly.jpg" ; class="imagenes">
+                </div>
+                <div class="centerVideo">
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/lrU5Z_6cMkA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen; class="videos"></iframe>
+                </div>
+            </dl>
+            <br>
+
+
             
         
         </main>
+        
 
     </body>
 
```

Comment:
Use Prettier with this file since it has a lot of empty spaces, these are bad for performance in code and readability